kirby/api/ext.py

```diff
@@ -37,13 +39,9 @@ class WebClientError(Exception):
 
 class Topic:
     def __init__(
-        self,
-        kirby_app,
-        topic_name_variable_name,
-        ssl_security_protocol=True,
-        testing=False,
+        self, kirby_app, topic_name, ssl_security_protocol=True, testing=False
```

Comment:
Usually `ssl_security_protocol` is called `use_tls`. I'd recommend sticking with that.

Reply:
I have renamed the parameter